metersink/api.py

```diff
@@ -20,29 +20,43 @@ def _init_logger():
 
 CONFIG = None
 
+def push_to_sinks(conf, data):
+    """puts received metering data to the configured billing sinks"""
+    # LOG.debug('%s', conf)
+    sinks = get_sinks(conf)
+    LOG.debug("pushing to sinks: %s", sinks)
+    for sink_type, sink_values in sinks.items():
+        if sink_type == "file":
+            for index, sink_name in enumerate(sinks["file"]["name"]):
+                # maybe we want to differ between events and polls here
+                # for now we put all incoming into the file
+                output_file(sink_name, data)
+        elif sink_type == "odoo":
+            odoo_handle(sinks['odoo'], conf, data)
+
 
 @app.route("/post_json", methods=["POST"])
 def process_json():
+    """Endpoint for json requests"""
     content_type = request.headers.get("Content-Type")
     if content_type == "application/json":
```

Comment:
I would invert the logic here and bail first. Then the "meat" of the function need not be indented.

Reply:
you are right. good idea!

 done.